### ISSUE TITLE
Subscriptions

### ISSUE THREAD

**Issue by ForestRingGames** [robotmedia/RMStore#231]
Hey! 

We've seen that there haven't been any updates for quite some time. Does it still work on iOS 13?
Does it support subscriptions still on the latest iOS?

We'd love to use RMStore as it has worked out perfectly for us in the past.

**Comment by istornz** [comment]
Hello, yes it's work perfectly on iOS 13 (tested on iPhone & iPad).

**Comment by ForestRingGames** [comment]
@istornz awesome! Have you also tested subscriptions? :)

**Comment by istornz** [comment]
@ForestRingGames Yes I have an auto renewable subscription & consumable product. Both works 👍
